### PR TITLE
feat(mobile-exp): Move multiple screenshots buttons to top

### PR DESCRIPTION
**Pull request by narsaynorath**
Moves the previous/next buttons above the screenshot.

In the Issue Details page it looks like:
Screen Shot 2022-11-15 at 2 48 54 PM

Opening the modal in the issue details page allows you to go prev/next:
Screen Shot 2022-11-15 at 2 49 50 PM

The modal in the attachments tab allows for pagination, but does not have the "X of Y" text:
Screen Shot 2022-11-15 at 2 50 27 PM

### CODE REVIEW

File: static/app/components/events/eventTagsAndScreenshot/screenshot/modal.tsx [hunk 1]
```diff
@@ -103,19 +105,65 @@ function Modal({
...
   const links = pageLinks ? parseLinkHeader(pageLinks) : undefined;
-  const previousDisabled =
-    links?.previous?.results === false && currentAttachmentIndex === 0;
-  const nextDisabled =
-    links?.next?.results === false &&
-    currentAttachmentIndex === MAX_SCREENSHOTS_PER_PAGE - 1;
+
+  // Pagination behaviour is different between the attachments tab with page links
+  // vs the issue details page where we have all of the screenshots fetched already
+  const paginationProps: ComponentProps<typeof ScreenshotPagination> | null = links
```
Comment (shruthilayaj) [inline static/app/components/events/eventTagsAndScreenshot/screenshot/modal.tsx:115]: nit: can we make this an if/else block so its more readable?

File: static/app/components/events/eventTagsAndScreenshot/index.tsx [hunk 2]
```diff
@@ -115,16 +117,29 @@ function EventTagsAndScreenshots({
...
         <div>
           <ScreenshotWrapper>
             <ScreenshotDataSection
               data-test-id="screenshot-data-section"
               title={
                 screenshots.length > 1
-                  ? tct('[current] of [total] [link:screenshots]', {
-                      current: screenshotInFocus + 1,
-                      total: screenshots.length,
+                  ? tct('[link:Screenshots]', {
```
Comment (shruthilayaj) [inline static/app/components/events/eventTagsAndScreenshot/index.tsx:142]: Can we sneak in this change in here as well? Making this and the screenshot link in line 145 blue so it's obvious it's clickable? 
Screen Shot 2022-11-16 at 11 11 26 AM

**Review by shruthilayaj — APPROVED** [review]
looks great!

File: static/app/components/events/eventTagsAndScreenshot/screenshot/modal.tsx (hunk 1 above)
Comment (narsaynorath) [reply static/app/components/events/eventTagsAndScreenshot/screenshot/modal.tsx]: https://github.com/getsentry/sentry/pull/41337/commits/0aa8a5a555957ce6866a687653d7e5dbc199b687

Done 👍

File: static/app/components/events/eventTagsAndScreenshot/index.tsx (hunk 2 above)
Comment (narsaynorath) [reply static/app/components/events/eventTagsAndScreenshot/index.tsx]: https://github.com/getsentry/sentry/pull/41337/commits/6ae9e58f1eb9594b55a69ef29a65373aee8c06b5

Added! I also simplified the screenshot/screenshots ternary for the title prop by just using `tn()` inside the link

**Pull request merged**